src/js/home/SubscribePanel.jsx

```diff
@@ -198,6 +218,13 @@ export default function SubscribePanel({
       ) : (
         <LoginMessage actionText={t("subscribe.loginAction")} />
       )}
+      {messageBox && (
+        <Modal {...messageBox} onClose={() => setMessageBox(null)}>
+          {messageBox.body.split("\n").map((p, i) => (
+            <p key={i}>{p}</p>
+          ))}
+        </Modal>
+      )}
```

Comment:
Can we put this into a render function, `renderMessageBox(messageBox, () => setMessageBox(null))` and reuse it